docs/getting_started.rst

```diff
@@ -46,6 +46,40 @@ If you have installation problems, consider opening an
 `issue <https://github.com/rl-institut/oemof-B3/issues>`_.
 
 
+How to install geopandas under Windows
+--------------------------------------
+The installation of geopandas on Windows can be challenging. According to the geopandas documentation (https://geopandas.org/getting_started/install.html) there are multiple ways to install it. It is recommended to use one of the following ways:
+
+1) Using the conda-forge channel
+
+Simply type
+
+::
+
+    conda install --channel conda-forge geopandas
+
+in the Anaconda prompt
+
+
+2) Creating a new environment
```

Comment:
Is using an environment really an alternative, or should it be rather our default advice?

Reply:
I agree with you that it should be our default advice. I tested both ways. Both lead to successfull installations of geopandas on windows for me, whereas with the second way I can't understand what the commands exactly do.

I noticed that you can not use 
```
    conda install python=3 geopandas
```

without 
```
    conda config --env --add channels conda-forge

    conda config --env --set channel_priority strict
```
before.

So I would suggest to say that it is recommended for windows users to install conda in a clean environment either with

```
    conda install --channel conda-forge geopandas
```
or:
```
    conda config --env --add channels conda-forge

    conda config --env --set channel_priority strict

    conda install python=3 geopandas
```

What do you think @jnnr ?

Reply:
I simplified the explanation to the installation with `conda-forge channel` in commit https://github.com/rl-institut/oemof-B3/pull/89/commits/adb9bb5162b003a51191eb10b82a46cf2915e230.